frontend/app/factories.js

```diff
@@ -70,32 +81,32 @@ angular.module('TicTacToe.factories', [])
                              available: false,
                              boards: []});
                 if (data[18] & ~data[19] & bitfield) {
-                service.data[row][col].player = 'x';
+                    service.data[row][col].player = 'x';
```

Comment:
is the unboxData method getting used now? in the PR that didn't made it, i removed it since it wasn't used at all.